### DISCUSSION TITLE
10 Query Limit

### DISCUSSION THREAD

**Discussion by karaeren** [TanStack/query#5697] [Q&A]
Is there a limit (or a way to limit queries with configuration) on the number of queries? I'm facing a problem where the first 10 queries are fetched, and the rest are stuck at loading state with null data. If I use the "Refetch" action on the 11th query, it fetches successfully.

I've tested with many query counts but it's always the first 10 that goes and the rest are stuck. I've also tested it on a coworker's machine and it's the same.

**Comment by TkDodo** [comment]
there is no limit on our end. maybe it's the API you're querying? Try replacing the QueryFunction with a simple `Promise.resole(jsonDataHere)`
